lib/cdo/shared_constants/curriculum/shared_course_constants.rb

```diff
@@ -78,37 +78,33 @@ module SharedCourseConstants
     ).freeze
 
     # The curriculum types used in curriculum quick assign
-    COURSE_OFFERING_CURRICULUM_TYPES = OpenStruct.new(
```

Comment:
when i grepped for this in dashboard, i got this:

```
../dashboard/app/models/course_offering.rb:  validates :curriculum_type, acceptance: {accept: Curriculum::SharedCourseConstants::COURSE_OFFERING_CURRICULUM_TYPES.to_h.values, message: "must be one of the course offering curriculum types. Expected one of: #{Curriculum::SharedCourseConstants::COURSE_OFFERING_CURRICULUM_TYPES.to_h.values}. Got: \"%{value}\"."}
```

which it looks like is what's breaking the ci